airflow/cli/commands/fastapi_api_command.py

```diff
@@ -81,16 +82,22 @@ def fastapi_api(args):
             str(args.hostname),
         ]
 
+        # There is no way to pass the apps to airflow/api_fastapi/main.py in the debug mode
+        # because fastapi dev command does not accept any additional arguments
+        # so environment variable is being used to pass it
+        os.environ["AIRFLOW_API_APPS"] = apps
```

Comment:
Instead of doing this (adding and then poping), you can just add the variable to the command like with `AIRFLOW_API_APPS=apps` in the command line arguments I think. 

Both are fine, not blocking.

Reply:
Going to keep the same for now, might revisit later. Created a draft issue: https://github.com/orgs/apache/projects/405/views/1?pane=issue&itemId=83765339

> Instead of doing this (adding and then poping), you can just add the variable to the command like with `AIRFLOW_API_APPS=apps` in the command line arguments I think.
> 
> Both are fine, not blocking.